charts/erigon/README.md

```
@@ -214,3 +230,12 @@
 ## Contributing
 
 We welcome and appreciate your contributions! Please see the [Contributor Guide](/CONTRIBUTING.md), [Code Of Conduct](/CODE_OF_CONDUCT.md) and [Security Notes](/SECURITY.md) for this repository.
+  # Option B: LoadBalancer
```

markdownlint-cli2 (0.17.2)

Headings must start at the beginning of the line

(MD023, heading-start-left)